app/src/store/modules/campaigns.js

```diff
@@ -129,7 +129,7 @@ export default {
     },
     async donate(_, { id, value }) {
       const userWallet = getWallet();
-      await userWallet.sendTransaction({
+      return await userWallet.sendTransaction({
```

Comment:
`await` is redundant here, just return the promise:

```suggestion
      return userWallet.sendTransaction({
```